app/controllers/index_controller.rb

```diff
@@ -4,8 +4,26 @@ def index
     @news = News.last(5)
     @facilities = Facility.all
     @workorder = Workorder.last(5)
-    if current_user.role == 'janitor'
-      redirect_to '/workorders'
+    @buildings = Building.where(user_id: current_user)
+    if current_user.role == 'admin'
```

Comment:
I would create a method on the user model, like this:
```ruby
def admin? 
  self.role == 'admin*
end
```  

then, in your controller, you can do:
```ruby
if current_user.admin?
 ...
```